sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -9094,6 +9408,139 @@ prefetch(AccessorT acc, PropertyListT props = {}) {
+/// @param SurfaceWidth is the surface width minus 1 in bytes
+/// @param SurfaceHeight is the surface height minus 1 in rows
+/// @param SurfacePitch is the surface pitch minus 1 in bytes
+/// @param X is zero based X-coordinate of the left upper rectangle corner in
+/// number of elements.
+/// @param Y is zero based Y-coordinate of the left upper rectangle corner in
+/// rows.
+/// @param props The optional compile-time properties. Only cache hint
+/// properties are used.
+/// @return is a vector of type T and size N, where N is
+///  BlockWidth * BlockHeight * NBlocks, if not transformed;
+///  otherwise,
+///  N = roundUpNextMultiple(BlockHeight, 4 / sizeof(T)) *
+///   getNextPowerOf2(BlockWidth) * NBlocks
+///
+template <typename T, int BlockWidth, int BlockHeight = 1, int NBlocks = 1,
+          bool Transposed = false, bool Transformed = false,
+          typename PropertyListT = oneapi::experimental::empty_properties_t,
+          int N = detail::get_lsc_block_2d_data_size<
+              T, NBlocks, BlockHeight, BlockWidth, Transposed, Transformed>()>
+__ESIMD_API std::enable_if_t<
+    ext::oneapi::experimental::is_property_list_v<PropertyListT>, simd<T, N>>
+load_2d(const T *Ptr, unsigned SurfaceWidth, unsigned SurfaceHeight,
+        unsigned SurfacePitch, int X, int Y, PropertyListT props = {}) {
+  return detail::load_2d_impl<T, BlockWidth, BlockHeight, NBlocks, Transposed,
+                              Transformed, PropertyListT>(
+      Ptr, SurfaceWidth, SurfaceHeight, SurfacePitch, X, Y);
+}
+
+/// template <typename T, int BlockWidth, int BlockHeight = 1, int NBlocks = 1,
+///          typename PropertyListT = empty_properties_t,
+///          int N = detail::get_lsc_block_2d_data_size<
+///              T, NBlocks, BlockHeight, BlockWidth, false, false>()>
+/// void
+/// prefetch_2d(const T *Ptr, unsigned SurfaceWidth, unsigned SurfaceHeight,
+///            unsigned SurfacePitch, int X, int Y, PropertyListT props = {});
+/// 2D USM pointer block prefetch.
+/// Supported platforms: PVC
+/// VISA instruction: lsc_load_block2d.ugm
+///
+/// Prefetches elements located at specified address.
+///
+/// @tparam T is element type.
+/// @tparam BlockWidth is the block width in number of elements.
+/// @tparam BlockHeight is the block height in number of elements.
+/// @tparam NBlocks is the number of blocks.
+/// @tparam N is the data size
+/// @param Ptr is the surface base address for this operation.
+/// @param SurfaceWidth is the surface width minus 1 in bytes
+/// @param SurfaceHeight is the surface height minus 1 in rows
+/// @param SurfacePitch is the surface pitch minus 1 in bytes
+/// @param X is zero based X-coordinate of the left upper rectangle corner
+/// in number of elements.
+/// @param Y is zero based Y-coordinate of the left upper rectangle corner
+/// in rows.
+/// @param props The optional compile-time properties. Only cache hint
+/// properties are used.
```

Comment:
It is not `optional` for `prefetch`, right?
```suggestion
/// @param props The compile-time properties that must specify cache hints.
```